website/content/en/docs/best-practices/multi-tenancy.md

```diff
@@ -0,0 +1,189 @@
+complete `InstallPlan` will fail on a cluster, which does not have these CRD/the Prometheus operator installed. 
+In this case, you might want to ensure the dependency requirement with [OLM dependency][olm-dep] or make clear 
+its requirement for the Operator consumers. 
+- **SHOULD** allow admins to customize the `requests/limits` amounts defined for the `Pod/Deployment`
+created by the Operator and not keep them fixed on the code.
+- **SHOULD** document how your Operator consumer can customize/rightsize the resources requests and limits for the 
+Operator and Operands `Pod/Deployments` or let its consumers know that the solution could adjust a better approach where 
+the Operator automatically adjust the values to the environment rather than asking its consumers to amend them
+
+You might also consider leveraging the [Vertical Pod Autoscaler][k8s-vertical-pod-scaling] to have the resources requested by the Operator 
+automatically adjusted to the cluster where it is deployed. You might also 
+look at allowing [horizontal pod autoscaling][k8s-autoscaling] for the other `Pod/Deployments` created by your Operator. 
+In this case, be aware that resource requests are also required for horizontal pod autoscaling to work.
+
+**NOTE:** If you are using OpenShift as Kubernetes distribution you might want to check the 
+doc [Automatically adjust pod resource levels with the vertical pod autoscaler
+][ocp-pod-scaling]. Caveat: If the VPA CRD is not available in the cluster where the operator 
+gets deployed the InstallPlan will fail. Please, see how to work with [OLM dependency][olm-dep] 
+if your project integrating with it.
+
+### Why should you set these?
+
+#### Resource Requests
+
+What happens when the resource requests are not set?
+
+- configurations made by the cluster administrators such as [ResourceQuota][k8s-resources-quotas] might not work without LimitRanges. The LimitRanger admission controller can provide default values for resource requests and limits when they have not been defined.
+- the Operators consumers might face resource shortages on a node when resource usage increases, for example, during a daily peak in request rate.
+- the Operators consumers might be unable to successfully deploy the Operator because it does not have the minimal resources available.
+- the scheduler cannot make an informed placement choice when it picks the nodes the operator pods will be running on
+- when there is memory contention on the node the pod is likely to get either evicted or OOM killed
+- when there is CPU contention on the node the pod is likely to get starved of CPU cycles making the operator unresponsive
+
+#### Resource Limits
+
+What happens when the resource limits are not set?
+
+Wrong configurations or code implementations can consume all resources available, affecting other components on cluster. 
+Also, it might let the containers more vulnerable such as to Dos Attacks. [More info][docker-security]. See that when there is memory 
```

Comment:
```suggestion
Also, it might leave the containers more vulnerable such as to Dos Attacks. [More info][docker-security]. See that when there is memory 
```